shared/src/test/scala/FutureBehavior.scala

```diff
@@ -28,11 +28,11 @@ class FutureBehavior extends munit.FunSuite {
           assert(false); 1
         }
         val res = c
-          .alt(Future {
+          .or(Future {
             val res = a.await + b.await
             res
           })
-          .alt(c)
+          .or(c)
```

Comment:
orElse is a good name too😂

Reply:
I hesitated with `orElse` because it implies a by-name argument (similar to `Option` and `Try`) while this is not by-name